lib/ui/prompt.js

```diff
@@ -76,9 +76,11 @@ PromptUI.prototype.processQuestion = function (question) {
 };
 
 PromptUI.prototype.fetchAnswer = function (question) {
+  var screen = this.createScreen();
   var Prompt = this.prompts[question.type];
-  var prompt = new Prompt(question, this.rl, this.answers);
+  var prompt = new Prompt(question, screen, this.rl, this.answers);
```

Comment:
@SBoudrias Before I start updating the tests, I wanna make sure we're on the same page. Happy with this?